vscode/src/codeActions.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ILanguageService } from "qsharp-lang";
+import * as vscode from "vscode";
+import { ICodeAction } from "../../npm/qsharp/lib/web/qsc_wasm";
+import { toVscodeWorkspaceEdit } from "./common";
+
+export function createCodeActionsProvider(languageService: ILanguageService) {
+  return new QSharpCodeActionProvider(languageService);
+}
+
+class QSharpCodeActionProvider implements vscode.CodeActionProvider {
+  constructor(public languageService: ILanguageService) {}
+  async provideCodeActions(
+    document: vscode.TextDocument,
+    range: vscode.Range | vscode.Selection,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I haven't checked out the branch to test, but I _think_ you can just remove these two params if not used, rather than suppress the warnings.

Reply:
Yes, it worked. Thank you!